lib/vmpooler/pool_manager.rb

```diff
@@ -189,7 +189,7 @@ def clone_vm(template, folder, datastore, target)
 
         # Generate a randomized hostname
         o = [('a'..'z'), ('0'..'9')].map(&:to_a).flatten
-        vm['hostname'] = o[rand(25)] + (0...14).map { o[rand(o.length)] }.join
+        vm['hostname'] = $config[:config]['prefix'] + o[rand(25)] + (0...14).map { o[rand(o.length)] }.join
```

Comment:
Yes, I figure different users might have different conventions (underscore, dash, colon, etc) and didn't want to make the assumption for them.